eth_tester/backends/pyevm/main.py

```diff
@@ -700,15 +702,18 @@ def _get_normalized_and_signed_evm_transaction(
         return evm_transaction.as_signed_transaction(signing_key)
 
     def _create_type_aware_unsigned_transaction(self, normalized_txn):
-        if all(_ in normalized_txn for _ in ("access_list", "gas_price")):
+        if all(_ in normalized_txn for _ in ("accessList", "gasPrice")):
             return self.chain.get_transaction_builder().new_unsigned_access_list_transaction(  # noqa: E501
-                **normalized_txn
+                **dict_keys_to_snake_case(normalized_txn)
```

Comment:
Yeah, I'm not convinced we can ignore my comment above. It leads to many calls to this method throughout the backend.